src/common/utils.go

```diff
@@ -296,8 +300,43 @@ func WriteTgzArchive(srcDirPath string, destFilePath string) error {
 	return nil
 }
 
-// GetNextMajorVersion computes next major version for a given one
-// for example, for 1.10.3 it's 2
+// CompressGzip compresses specified file  with gzip.BestCompression level
+func CompressGzip(srcFilePath string, destFilePath string) error {
+	var err error
+
+	// src file reader
+	srcFileReader, err := os.Open(srcFilePath)
+	if err != nil {
+		return fmt.Errorf("Failed to open source file %s: %s", srcFilePath, err)
+	}
+
+	defer srcFileReader.Close()
+
+	// dest file writer
+	destFile, err := os.Create(destFilePath)
```

Comment:
If file with such name already exists does it produce an error?

Reply:
If file already exists, it will be truncated. I'm not sure that this function should make a decision about file existent.

Reply:
May be some flags should be passed

Reply:
I don't want to make this function more difficult.

BTW, when you run `cartridge pack tgz` two times without changing a version, it rewrites existing TGZ archive with a new one.

Reply:
Ok - rewrite ok